core/shared/src/main/scala/zio/prelude/ZValidation.scala

```diff
@@ -85,6 +85,15 @@ sealed trait ZValidation[+W, +E, +A] { self =>
       case Success(_, a) => success(a)
     }
 
+  /**
+   * Returns the value of the log.
+   */
+  final def getLog: Chunk[W] =
```

Comment:
`log`? :smile_cat:

Reply:
We already use `log` for writing to the log. We could overload it. I thought that might be a little confusing but I definitely thought about it.

Reply:
Well those would be options so I think we already have `toOption` which does the same thing got the value channel and then you could fold for the error case or maybe we could have `errorsOption` or something.

Reply:
Scratch that :point_up: I hadn't read your comment above.